spec/models/course_spec.rb

```diff
@@ -11,8 +11,8 @@
 #  qualification           :integer          not null
 #  start_date              :datetime
 #  study_mode              :text
-#  accrediting_provider_id :integer
-#  provider_id             :integer          default(0), not null
+#  accrediting_course_id :integer
+#  course_id             :integer          default(0), not null
```

Comment:
what are these changes related to?

Reply:
I think this was brought in by a search/replace. Good spot, will regenerate the annotations.